cmake/FindSDL2.cmake

```diff
@@ -0,0 +1,163 @@
+# Locate SDL2 library
```

Comment:
If you're using "disabled-mode" modules it make no sense to add this file. You can move this code to `cmake/Hunter/disabled-mode/SDL2Config.cmake`.